### PR TITLE
OSD-8883: added cvo gates for cluster updates

### PR DESCRIPTION
**Pull request by ramonbutter**
syncing admin-acks to osd clusters for the case it is specified in the ClusterDeployment annotation

### CODE REVIEW

File: deploy/osd-cluster-acks/4.9/config.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+deploymentMode: "SelectorSyncSet" 
+selectorSyncSet:
+  matchExpressions:
+  - key: hive.openshift.io/version-major/minor
```
Comment (rafael-azevedo) [inline deploy/osd-cluster-acks/4.9/config.yaml:4]: Should be 

  - key: hive.openshift.io/version-major-minor

File: hack/00-osd-managed-cluster-config-integration.yaml.tmpl [hunk 2]
```diff
@@ -5523,6 +5523,36 @@ objects:
...
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: osd-cluster-acks-4.9
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+      matchExpressions:
+      - key: hive.openshift.io/version-major/minor
```
Comment (rafael-azevedo) [inline hack/00-osd-managed-cluster-config-integration.yaml.tmpl:5539]: will have to re-run make to fix this

**Review by rafael-azevedo — CHANGES_REQUESTED** [review]

File: hack/00-osd-managed-cluster-config-integration.yaml.tmpl (hunk 2 above)
Comment (jharrington22) [reply hack/00-osd-managed-cluster-config-integration.yaml.tmpl:5539]: There is a github action that would do it but it wont update until @ramonbutter updates the PR :)

**Review by rafael-azevedo — APPROVED** [review]

**Pull request merged**